build/buildkit/buildimage/image_interface.go

```diff
@@ -35,3 +35,8 @@ type ImageSetter interface {
 	// Set :fill up v1.image struct, like image annotations, platform and so on.
 	Set(*v1.Image) error
 }
+
+type Shadow interface {
```

Comment:
What Shadow means?

Reply:
Some operations are not reflected form the kubefile, or the build cmd, like we support download images form ”imageListWithAuth.yaml “.

Reply:
Just add a `imageListWithAuth` processor